README.md

```diff
@@ -51,8 +51,11 @@ yarn install
 
 You can then use the [Gatsby CLI commands](https://www.gatsbyjs.com/docs/reference/gatsby-cli/) `build`, `clean`, `develop` and `serve`.
 
-To build the website, run
+Our custom source plugin fetches information from GitHub, and therefore needs a GitHub [personal access token] in the environment named `GITHUB_TOKEN`. 
+
+[personal access token]: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
```

Comment:
Do we want to include information about how to update this in the repository settings here or is the comparable explanation in the [engine wiki](https://github.com/MovingBlocks/Terasology/wiki/Maintenance#how-to-fix-an-expired-github-action-token) sufficient if linked?

Reply:
I think the link would be sufficient.